src/executor/attach_op_execs_pass.cc

```diff
@@ -153,12 +158,15 @@ class StatefulComputeExecutor : public StorageFallbackOpExecutor {
 
 
 // stateful compute_ex executor
-class StatefulComputeExExecutor : public OpExecutor {
+class StatefulComputeExExecutor : public MKLDNNOpExecutor {
```

Comment:
Making this class derived from MKLDNNOpExecutor while it's only valid when MXNET_USE_MKLDNN == 1. This violates the inheritance class is-a relationship